runtime/near-vm-logic/src/tests/miscs.rs

```diff
@@ -876,70 +876,194 @@ fn test_ed25519_verify() {
 
     let bad_signature: [u8; 64] = [1; 64];
 
+    let mut forged_signature = signature.clone();
+    // create a forged signature with the `s` scalar not properly reduced
+    // https://docs.rs/ed25519/latest/src/ed25519/lib.rs.html#302
+    forged_signature[63] = 0b1110_0001;
+
     let public_key: [u8; 32] = [
         32, 122, 6, 120, 146, 130, 30, 37, 215, 112, 241, 251, 160, 196, 124, 17, 255, 75, 129, 62,
         84, 22, 46, 206, 158, 184, 57, 224, 118, 35, 26, 182,
     ];
+    let mut forged_public_key = public_key.clone();
+    // create a forged public key to force a PointDecompressionError
+    // https://docs.rs/ed25519-dalek/latest/src/ed25519_dalek/public.rs.html#142
+    forged_public_key[31] = 0b1110_0001;
```

Comment:
Let's create a separate `ed25519_verify` test module for this test. And maybe split it into two: one `#[test]` to verify behavior and errors, and one `#[test]` to check registers in particular.